packages/tizen_app_manager/tizen/src/application_utils.cc

```diff
@@ -4,145 +4,194 @@
 
 #include "application_utils.h"
 
-namespace application_utils {
-
-bool ExtractValueFromMap(const flutter::EncodableValue &arguments,
-                         const char *key, std::string &out_value) {
-  if (std::holds_alternative<flutter::EncodableMap>(arguments)) {
-    flutter::EncodableMap map = std::get<flutter::EncodableMap>(arguments);
-    auto iter = map.find(flutter::EncodableValue(key));
-    if (iter != map.end() && !iter->second.IsNull()) {
-      if (auto pval = std::get_if<std::string>(&iter->second)) {
-        out_value = *pval;
-        return true;
-      }
-    }
-  }
-  return false;
-}
-
-bool AppMetaDataCB(const char *key, const char *value, void *user_data) {
-  if (key != nullptr) {
-    flutter::EncodableMap *metadata = (flutter::EncodableMap *)user_data;
-    metadata->insert(
-        std::pair<flutter::EncodableValue, flutter::EncodableValue>(
-            std::string(key), std::string(value)));
-    LOG_INFO("AppMetaDataCB key: %s, value: %s", key, value);
-  }
-  return true;
-}
-
-int GetAppData(app_info_h app_info, flutter::EncodableMap &value) {
+namespace {
+char *GetAppId(app_info_h app_info, flutter::EncodableMap &value) {
```

Comment:
Can you read [this document](https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace) about the vertical white space and apply it to this plug-in as a whole?
I think If you look at other plug-in implementations, you'll get help.

Reply:
> Blank lines immediately inside a declaration of a namespace or block of namespaces may help readability by visually separating the load-bearing content from the (largely non-semantic) organizational wrapper. Especially when the first declaration inside the namespace(s) is preceded by a comment, this becomes a special case of the previous rule, helping the comment to "attach" to the subsequent declaration.

I added a blank line just below the namespace. thank you!